.gitignore

```diff
@@ -13,6 +13,7 @@ TestResults
 *.user
 *.sln.docstates
 *.DS_Store
+*.yaml
```

Comment:
Dunno if this might interfere with being able to upload/edit/clone YAML files that we want shared, like Travis or containers. May want to assign the ignore to just the tools subdirectory for now.

Reply:
Updated to ignore just this file for now.